### PR TITLE
Applied theme variable for hiding causal grid on zurich action page

### PR DESCRIPTION
**Pull request by lilia1891**

### CODE REVIEW

File: src/pages/actions/[slug].tsx [hunk 1]
```diff
@@ -120,6 +121,7 @@ export default function ActionPage() {
   const activeGoal = useReactiveVar(activeGoalVar);
   const site = useSite();
   const [activeSubAction, setActiveSubAction] = useState(undefined);
+  const theme = useContext(ThemeContext);
```
Comment (woodwoerk) [inline src/pages/actions/[slug].tsx:124]: You can use `useTheme` here, it's a bit shorter and the name has a clearer intention. [Here's](https://github.com/kausaltech/kausal-paths-ui/blob/main/src/components/Layout.tsx#L58) an example.

File: src/pages/actions/[slug].tsx [hunk 2]
```diff
@@ -280,7 +282,7 @@ export default function ActionPage() {
           />
         </Container>
       )}
-      {causalNodes.length > 0 && (
+      {causalNodes.length > 0 && !theme?.settings.hideActionGrid && (
```
Comment (woodwoerk) [inline src/pages/actions/[slug].tsx:285]: Also when using `useTheme` the value will always be defined, so the `?` won't be needed.

File: src/pages/actions/[slug].tsx (hunk 1 above)
Comment (lilia1891) [reply src/pages/actions/[slug].tsx]: Done, thanks!

**Review by woodwoerk — APPROVED** [review]
Looks good!

**Pull request merged**